brvs-core/src/main/java/iroha/validation/transactions/plugin/impl/SoraDistributionPluggableLogic.java

```diff
@@ -0,0 +1,427 @@
+/*
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package iroha.validation.transactions.plugin.impl;
+
+import static iroha.validation.utils.ValidationUtils.advancedQueryAccountDetails;
+
+import iroha.protocol.Commands.Command;
+import iroha.protocol.Commands.TransferAsset;
+import iroha.protocol.Endpoint.TxStatus;
+import iroha.protocol.TransactionOuterClass.Transaction;
+import iroha.validation.transactions.plugin.PluggableLogic;
+import iroha.validation.utils.ValidationUtils;
+import java.math.BigDecimal;
+import java.math.MathContext;
+import java.math.RoundingMode;
+import java.security.KeyPair;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import jp.co.soramitsu.iroha.java.IrohaAPI;
+import jp.co.soramitsu.iroha.java.QueryAPI;
+import jp.co.soramitsu.iroha.java.TransactionBuilder;
+import jp.co.soramitsu.iroha.java.Utils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.StringUtils;
+
+/**
+ * Sora distribution logic processor
+ */
+public class SoraDistributionPluggableLogic
+    extends PluggableLogic<Transaction, Map<String, BigDecimal>> {
+
+  private static final Logger logger = LoggerFactory
+      .getLogger(SoraDistributionPluggableLogic.class);
+  private static final String COMMA_SPACES_REGEX = ",\\s*";
+  public static final String DISTRIBUTION_PROPORTIONS_KEY = "distribution";
+  public static final String DISTRIBUTION_FINISHED_KEY = "distribution_finished";
+  private static final String XOR_ASSET_ID = "xor#sora";
+  private static final MathContext XOR_MATH_CONTEXT = new MathContext(18, RoundingMode.DOWN);
+  private static final int TRANSACTION_SIZE = 9999;
+  private static final String DESCRIPTION_FORMAT = "Distribution from %s";
+  private static final String EMPTY_DETAIL_RESPONSE = "{}";
```

Comment:
It looks like unused field

Reply:
Yes, I actually pushed the commit with its removal to the wrong remote 😄 Thanks